blogs/views.py

```diff
@@ -0,0 +1,43 @@
+from django.urls import reverse_lazy, reverse
+from django.views.generic import ListView, DetailView, CreateView, UpdateView, DeleteView
+from .models import Post
+
+class PostsListView(ListView):
+    model = Post
+    template_name = 'blogs/home_data.html'
+    context_object_name = 'posts'
+
+    def get_queryset(self):
+        """ Выводим только опубликованные статьи """
+        return Post.objects.filter(is_published=True)
```

Comment:
nit: лучше переопределять родительский метод
```suggestion
        qs = super().get_queryset()
        return qs.filter(is_published=True)
```